tests/conftest.py

```diff
@@ -467,6 +467,8 @@ def initial_data(logged_on_user, users_fixture, streams_fixture):
         'last_event_id': -1,
         'muted_topics': [],
         'realm_user_groups': [],
+        'zulip_version': '2.2-dev',
+        'zulip_feature_level': 1,
```

Comment:
This could be a good place to start exploring how we might use parametrization to encode variations per feature level and/or version. That is, our test data (and expectations) varies if the server covers a range of versions. This is already the case here, as the code is testing for the presence of `zulip_feature_level`, which we're not testing being present/absent.